lib/wallet/test/unit/Cardano/Wallet/Shelley/TransactionSpec.hs

```diff
@@ -1198,19 +1199,21 @@ prop_sealedTxRecentEraRoundtrip
     sealedTxB = sealedTxFromBytes' currentEra txBytes
 
 makeShelleyTx
-    :: IsRecentEra era
+    :: Write.IsRecentEra era
```

Comment:
Why use `IsRecentEra` qualified but not `RecentEra`?

Reply:
Historical reasons only. `RecentEra` was unqualified before I touched the file; I at least wanted to qualify all `IsRecentEra` that I touched.